standard/standard/clause_4_terms_and_definitions.adoc

```diff
@@ -1,8 +1,16 @@
 == Terms and Definitions
-This document uses the terms defined in Sub-clause 5.3 of [OGC 06-121r8], which is based on the ISO/IEC Directives, Part 2, Rules for the structure and drafting of International Standards. In particular, the word “shall” (not “must”) is the verb form used to indicate a requirement to be strictly followed to conform to this standard.
```

Comment:
Please use the default OGC boilerplate, do not use a custom one, we are trying to update to the latest OGC practices.

Reply:
Ok, fixed.